docs/docs/inputs_outputs.md

```diff
@@ -49,7 +74,7 @@ $ ffmpeg -i input_video.mp4 -c copy -map 0 -segment_time 15 -f segment -reset_ti
 `zamba` supports the same formats as FFMPEG, [which are listed here](https://www.ffmpeg.org/general.html#Supported-File-Formats_002c-Codecs-or-Features).
 
 Videos do not need to be resized to a particular resolution since `zamba`
- resizes every frame it uses to `404 x 720` pixels.
+resizes every frame it uses to `404 x 720` pixels.<!-- TODO: is that resizing still true?><!-->
```

Comment:
I'm don't think this happens automatically - it only happens when height and width are specified in the default `VideoLoaderConfig`, which they are not by default

Reply:
I think this may end up being per-model so should go on the details of that model:
https://github.com/drivendataorg/zamba-algorithms/blob/main/zamba_algorithms/models/configs/time_distributed.yaml#L1-L10